src/components/application_manager/test/commands/mobile/set_global_properties_test.cc

```diff
@@ -553,7 +563,7 @@ TEST_F(SetGlobalPropertiesRequestTest, Run_VRWrongOrder_Canceled) {
       .WillOnce((Return(mobile_apis::Result::SUCCESS)));
   EXPECT_CALL(app_mngr_, RemoveAppFromTTSGlobalPropertiesList(kConnectionKey));
   EmptyExpectationsSetupHelper();
-
+  EXPECT_CALL(app_mngr_, GetRPCService()).WillOnce(ReturnRef(rpc_service_));
```

Comment:
@ValeriiMalkov please move it to SetUp()

Reply:
@LuxoftAKutsan @LitvinenkoIra fixed in https://github.com/LitvinenkoIra/sdl_core/pull/3/commits/99eb6746366c2ae8bff7be8fdc67dc300f2c5330